### DISCUSSION TITLE
Following tutorial #3: no protein expression database available

### DISCUSSION THREAD

**Discussion by learnCode-cpu** [Open-Systems-Pharmacology/Forum#1351] [Questions & Problems] [answered]
Hi, New to PK-Sim, following this tutorial. 
https://www.youtube.com/watch?v=8kd1RfK_ms4
I understand the the program layout has changed in the new version that I have. However, when I follow the steps to add CYP3A4 enzyme, the table that is generated has no values. When I hit 'database query', it says "No protein expression database available for species 'Human'." I tried other species but got the same result. 

Has anyone else encountered this or have a solution for this? I would appreciate your help. I am using the latest version of the s/w. Thanks.

**Comment by StephanSchaller** [comment, marked as answer]
Hi @learnCode-cpu , please refer to the manual: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile#editing-protein-expression-by-querying-the-expression-database

Also, you can use the esqLABS Ressources:
Industry: https://training-esqlabs.talentlms.com/ (some basic courses are free)
Academia: https://learn-esqlabs.talentlms.com/ (completely free)

**Comment by learnCode-cpu** [comment]
Perfect, thank you.
